### DISCUSSION TITLE
example data for water

### DISCUSSION THREAD

**Discussion by zhubonan** [deepmodeling/deepmd-kit#2052] [General]
I am wondering if there is a public repository containing the water AIMD data mentioned in: https://www.sciencedirect.com/science/article/pii/S0010465518300882#sec4.

I can see examples under `examples/water`, but according to the documentation it only has limited data points and not for production. 

Thanks in advance.

**Comment by njzjz** [comment]
It can be downloaded from https://dplibrary.deepmd.net//#/elements_list, although I'm not sure which project it is.

Note: this website is quite slow

**Reply by zhubonan** [reply]
Ahh I see thanks!
